paper-input-container.html

```diff
@@ -177,9 +176,10 @@
       }
 
       .input-content {
-        position: relative;
         @apply(--layout-horizontal);
         @apply(--layout-center);
+
+        position: relative;
```

Comment:
It is not necessary, but most of the elements have the external `@apply()`'s at the beginning.

I think it would be awesome to have a style guide of styles for the elements. :smiley: